### PR TITLE
Feature/return home button

### PR DESCRIPTION
**Pull request by mbenfowler**
## Description
This pr adds our return home functionality when either the home button or title is clicked
Features:
- Moved "What's Cookin" title to the top left in the nav bar
- Added home button to the left of search bar, move search area to center of nav
- add new settings button to the top right

## Testing Notes
- Styling for new elements look good
- When either element is clicked, return to current view with no filters enabled
- Any search value in search bar should also be removed
- I made some minor code style refactors in domUpdates as well, make sure that looks right

### CODE REVIEW

**Comment by mbenfowler** [comment]
[[FEEDBACK] Have a way to get back home if no recipes are found](https://trello.com/c/5I5upBRB/49-feedback-have-a-way-to-get-back-home-if-no-recipes-are-found)

File: src/domUpdates.js [hunk 1]
```diff
@@ -213,10 +214,12 @@ const addScrollBar = (element) => {
...
-  instructionSection.innerHTML = `<p>Directions</p>
-                                  <section class='instruction-steps'> 
-                                    ${instructions.join('')} 
-                                  </section>`;
+  instructionSection.innerHTML = `
+    <p>Directions</p>
+    <section class='instruction-steps'>
+      ${instructions.join('')}
+    </section>
+  `;
```
Comment (lauraguerra1) [inline src/domUpdates.js:222]: omg thank you for fixing this!

File: dist/index.html [hunk 2]
```diff
@@ -9,13 +9,14 @@
   </head>
   <body>
     <nav> 
+      <h1> What's cookin? </h1>
       <div class="nav-child">
+        <img role="button" class="nav-icon" id="homeBtn" src="home.png" alt="Return home">
```
Comment (lauraguerra1) [inline dist/index.html:14]: looks good and works as expected!

**Review by lauraguerra1 — APPROVED** [review]

File: src/domUpdates.js [hunk 3]
```diff
@@ -57,27 +57,26 @@ const createSingleRecipeHTML = singleRecipe => {
...
+      <article class="individual-recipe" id="${singleRecipe.id}">
+        <div class="recipe-image-div">
+          <img class="recipe-image"src="${singleRecipe.image}" alt="${singleRecipe.name}">
+          <div class="hover-card">
+            <h4>${singleRecipe.pitch}</h4>
+          </div>
+        </div>
+        <h2>${singleRecipe.name}</h2>
+      </article>
     </article>
```
Comment (taranveersingh93) [inline src/domUpdates.js:79]: sorry what exactly changed here? Indentation?

File: src/styles.css [hunk 4]
```diff
@@ -54,26 +48,33 @@ body {
...
+
+nav > h1 {
+  text-align: center;
+  margin-left: 25px;
+  cursor: pointer;
 }
 
 .nav-child {
   display: flex;
-  justify-content: space-around;
```
Comment (taranveersingh93) [inline src/styles.css:64]: justify-content was aligning the children elements of nav-child evenly. With this change, the nav has nav-child elements in flexbox but the nav-child elements don't have centered children (they're left-aligned) which could cause problem if something else were to change (I can not imagine the exact situation).
Could we centre the elements inside all nav-child?
I know it looks fine to the user right now but it just feels like a loose end for any future changes.

File: src/styles.css [hunk 5]
```diff
@@ -54,26 +48,33 @@ body {
...
+}
+
 .nav-icon {
   height: 1.75rem;
   background-color: white;
   opacity: 0.4;
   border-radius: 50%;
-  margin: 0 auto;
+  margin: 0 2px 0 2px;
+  padding: 2px;
```
Comment (taranveersingh93) [inline src/styles.css]: Could the padding please be increased for nav-icons? I tried 5px padding and it looks pleasing without looking bloated. Right now I personally felt that the container for a nav icon felt crowded.

File: src/styles.css [hunk 6]
```diff
@@ -54,26 +48,33 @@ body {
...
+}
+
+nav > h1 {
+  text-align: center;
+  margin-left: 25px;
+  cursor: pointer;
 }
 
 .nav-child {
   display: flex;
```
Comment (taranveersingh93) [inline src/styles.css:61]: Do you think we could give some spacing between home icon, searchbar, search icon? It would look super clean.

**Review by taranveersingh93 — CHANGES_REQUESTED** [review]
Saves us some real estate and looks great. 
Requested some changes.

File: src/domUpdates.js (hunk 3 above)
Comment (mbenfowler) [reply src/domUpdates.js:79]: just fixing up some indenting

File: src/styles.css (hunk 4 above)
Comment (mbenfowler) [reply src/styles.css:64]: sure i'll add this back then

File: src/styles.css (hunk 5 above)
Comment (mbenfowler) [reply src/styles.css]: done

File: src/styles.css (hunk 6 above)
Comment (mbenfowler) [reply src/styles.css:61]: done

File: src/domUpdates.js (hunk 3 above)
Comment (taranveersingh93) [reply src/domUpdates.js:79]: Thank you for your service

File: dist/index.html [hunk 7]
```diff
@@ -9,13 +9,17 @@
...
-      <div class="nav-child">
-        <input class="nav-icon" id="searchBar" type="search" placeholder="Search.." >
+      <h1> What's cookin? </h1>
+      <div class="nav-child" id="searchArea">
+        <img role="button" class="nav-icon" id="homeBtn" src="home.png" alt="Return home"> 
+        <input id="searchBar" type="search" placeholder="Search.." >
         <img role="button" class="nav-icon" id="searchBtn" src="./images/search-button.png" alt="Magnifying Glass Icon"> 
       </div>
+      <div class="nav-child">
+        <img role="button" class="nav-icon" id="settings" src="settings.png" alt="Return home">
```
Comment (lauraguerra1) [inline dist/index.html]: Should this alt be like settings button or something instead of return home?

**Review by taranveersingh93 — APPROVED** [review]
Sweet!

**Review by lauraguerra1 — CHANGES_REQUESTED** [review]

File: dist/index.html (hunk 7 above)
Comment (mbenfowler) [reply dist/index.html]: crap yah, i'll add an alt

Comment (mbenfowler) [reply dist/index.html]: done!

**Pull request merged**